pages/common/http.md

```diff
@@ -1,32 +1,36 @@
 # http
 
-> HTTPie: HTTP client, aims to be easier to use than cURL.
-> More information: <https://httpie.org>.
+> HTTPie: CLI tool designed for testing, debugging, and generally interacting with APIs & HTTP servers.
+> More information: <https://httpie.io/docs/cli/usage>.
 
-- Download a URL to a file:
+- Make a simple GET request (shows response header and content):
 
-`http --download {{example.org}}`
+`http {{https://example.org}}`
 
-- Send form-encoded data:
+- Print specific output content (`H`: request headers, `B`: request body, `h`: response headers, `b`: response body, `m`: response metadata):
```

Comment:
```suggestion
- Print specific output content (request [H]eaders, request [B]ody, response [h]eaders, response [b]ody, response [m]etadata):
```

Reply:
This syntax is used only for highlighting short options.
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#short-option-mnemonics

Reply:
But why dont use this syntax in this case?

Reply:
Because it is not an option itself, but a value to the option. The syntax is meant for short options.

Reply:
If we start using this for other things, it'll quickly get confusing. How will you know whether it denotes a short option or something else?